src/apps/backend/modules/task/internal/store/task-db.ts

```diff
@@ -20,7 +21,12 @@ export const TaskDbSchema: Schema = new Schema<TaskDB>(
       index: true,
       required: true,
     },
-    name: {
+    description: {
+      type: String,
+      index: true,
+      required: true,
+    },
+    title: {
       type: String,
       index: true,
```

Comment:
can we remove index here too? cause no reader method is performing search by title

Reply:
done with this in upcoming PR